### DISCUSSION TITLE
Initial data load and realtime

### DISCUSSION THREAD

**Discussion by aonghas** [pocketbase/pocketbase#4572] [Q&A] [answered]
Hi there 

Just wanted to sense check if I'm doing this correctly. If I want to set up realtime updates for a filtered collection, would I do a regular GET with the same filter first to get the initial data and _then_ call the realtime subscribe once the initial data has come back? Or should I call both simultaneously? Or is there a better approach? I know with firestore the subscribing itself returns the initial data in the snapshot callback so just wanted to know if pocketbase requires 2 separate calls for the same functionality? Thanks!

**Comment by ganigeorgiev** [comment, marked as answer]
PocketBase requires 2 separate calls. The realtime record subscriptions react only on changes (aka. create, update, delete operations).
